docs/extend/plugins_volume.md

```diff
@@ -164,3 +164,52 @@ this point.
 
 Respond with a string error if an error occurred.
 
+
+### /VolumeDriver.Get
+
+**Request**:
+```
+{
+    "Name": "volume_name"
+}
+```
+
+Get the volume info.
+
+
+**Response**:
+```
+{
+  "Name": "volume_name",
+  "Mountpoint": "/path/to/directory/on/host",
+  Err: ""
+}
+```
+
+Respond with a string error if an error occurred.
```

Comment:
Is that a plain string? Or a JSON response with `Err` containing the string?

Does the plugin API use headers to communicate errors (e.g. a 500 internal server error)?